dev/tasks/tasks.yml

```
@@ -927,6 +927,11 @@ tasks:
       - flight-sql-{no_rc_version}-tests.jar
       - flight-sql-{no_rc_version}.jar
       - flight-sql-{no_rc_version}.pom
+      - flight-sql-jdbc-driver-{no_rc_version}-javadoc.jar
+      - flight-sql-jdbc-driver-{no_rc_version}-sources.jar
+      - flight-sql-jdbc-driver-{no_rc_version}-tests.jar
+      - flight-sql-jdbc-driver-{no_rc_version}.jar
+      - flight-sql-jdbc-driver-{no_rc_version}.pom
```

As this was merged, can we update it to: https://github.com/apache/arrow/commit/bced07de986b9f7baaa4885d7f03f4e46d1abed8
```suggestion
      - flight-sql-jdbc-driver-{no_rc_snapshot_version}-javadoc.jar
      - flight-sql-jdbc-driver-{no_rc_snapshot_version}-sources.jar
      - flight-sql-jdbc-driver-{no_rc_snapshot_version}-tests.jar
      - flight-sql-jdbc-driver-{no_rc_snapshot_version}.jar
      - flight-sql-jdbc-driver-{no_rc_snapshot_version}.pom
```